src/locales/en-US/commands.json

```diff
@@ -1257,5 +1267,35 @@
     "commandUsage": "<code>",
     "missingCode": "You need to give me some code to upload!",
     "hereIsYourURL": "Here's your link!"
+  },
+  "connections": {
+    "commandDescription": "Shows all the accounts your account is connected to.",
+    "title": "Your connections",
+    "connected": "Connected as {{user}}",
+    "notConnected": "Not connected. Click here to connect."
+  },
+  "scrobbling": {
+    "commandDescription": "Sees information about audio scrobbling on last.fm with Switchblade.",
+    "title": "Audio scrobbling",
+    "notConnectedDescription": "On switchblade you can scrobble the songs you listen to your last.fm account. You can connect you account [here]({{link}}).",
```

Comment:
I think it would be correct as „On switchblade you can scrobble the songs you listen to **to** your last.fm account.”

Reply:
@Lireoy